doc/examples/features_detection/plot_scattering.py

```diff
@@ -0,0 +1,129 @@
+"""
+===================
+Scattering Features
+===================
+
+This example shows how to use the scattering transform on images. It explains how to access easily different scattering
+coefficients stored in the output vector. For more information on how to use scattering coefficients for image classification,
+please check the example *Scattering features for supervised learning*.
+
+We call the scattering features (or coefficients) the output of the scattering transform applied to an image. These features
+have shown to outperform any other 'non-learned' image representations for image classification tasks [2]_.
+Structurally, they are very similar to deep learning representations, with fixed filters which are Morlet filters, in this
+implementation.
+Since the scattering transform is based on the Discrete Wavelet transform (DWT), it is stable to deformations.
```

Comment:
Not sure how OK we are to say this, since it  kind of mimicks a continuous wavelet transform. CC @edouardoyallon